<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-8</y>
      <width>831</width>
      <height>601</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/background.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCustomers">
    <property name="title">
     <string>Customers</string>
    </property>
    <addaction name="actionCreate_Customer"/>
    <addaction name="actionView_Customers"/>
   </widget>
   <widget class="QMenu" name="menuEmployees">
    <property name="title">
     <string>Employees</string>
    </property>
    <addaction name="actionCreate_Employee"/>
    <addaction name="actionView_Employee"/>
   </widget>
   <widget class="QMenu" name="menuProducts">
    <property name="title">
     <string>Products</string>
    </property>
    <addaction name="actionView_Products"/>
    <addaction name="actionView_Products_2"/>
   </widget>
   <widget class="QMenu" name="menuOrders">
    <property name="title">
     <string>Orders</string>
    </property>
    <addaction name="actionCreate_Orders"/>
    <addaction name="actionView_Orders"/>
   </widget>
   <widget class="QMenu" name="menuSuppliers">
    <property name="title">
     <string>Suppliers</string>
    </property>
    <addaction name="actionCreate_Supplier"/>
    <addaction name="actionView_Supplier"/>
   </widget>
   <widget class="QMenu" name="menuDepartments">
    <property name="title">
     <string>Departments</string>
    </property>
    <addaction name="actionCreate_Department"/>
    <addaction name="actionView_Department"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>Reports</string>
    </property>
    <widget class="QMenu" name="menuCreate_report">
     <property name="title">
      <string>Create report</string>
     </property>
     <addaction name="actionRevenue_per_Customer"/>
     <addaction name="actionRevenue_per_Product"/>
     <addaction name="actionRevenue_per_Supplier"/>
    </widget>
    <addaction name="menuCreate_report"/>
    <addaction name="actionView_reports"/>
    <addaction name="actionPrint_report"/>
   </widget>
   <addaction name="menuCustomers"/>
   <addaction name="menuEmployees"/>
   <addaction name="menuProducts"/>
   <addaction name="menuSuppliers"/>
   <addaction name="menuOrders"/>
   <addaction name="menuDepartments"/>
   <addaction name="menuReports"/>
  </widget>
  <action name="actionCreate_Customer">
   <property name="text">
    <string>Create Customer</string>
   </property>
  </action>
  <action name="actionView_Customers">
   <property name="text">
    <string>View Customers</string>
   </property>
  </action>
  <action name="actionCreate_Employee">
   <property name="text">
    <string>Create Employee</string>
   </property>
  </action>
  <action name="actionView_Employee">
   <property name="text">
    <string>View Employee</string>
   </property>
  </action>
  <action name="actionView_Products">
   <property name="text">
    <string>Create Products</string>
   </property>
  </action>
  <action name="actionView_Products_2">
   <property name="text">
    <string>View Products</string>
   </property>
  </action>
  <action name="actionCreate_Orders">
   <property name="text">
    <string>Create Orders</string>
   </property>
  </action>
  <action name="actionView_Orders">
   <property name="text">
    <string>View Orders</string>
   </property>
  </action>
  <action name="actionCreate_Supplier">
   <property name="text">
    <string>Create Supplier</string>
   </property>
  </action>
  <action name="actionView_Supplier">
   <property name="text">
    <string>View Supplier</string>
   </property>
  </action>
  <action name="actionCreate_Department">
   <property name="text">
    <string>Create Department</string>
   </property>
  </action>
  <action name="actionView_Department">
   <property name="text">
    <string>View Department</string>
   </property>
  </action>
  <action name="actionView_reports">
   <property name="text">
    <string>View reports</string>
   </property>
  </action>
  <action name="actionPrint_report">
   <property name="text">
    <string>Print report</string>
   </property>
  </action>
  <action name="actionRevenue_per_Customer">
   <property name="text">
    <string>Revenue per Customer</string>
   </property>
  </action>
  <action name="actionRevenue_per_Product">
   <property name="text">
    <string>Revenue per Product</string>
   </property>
  </action>
  <action name="actionRevenue_per_Supplier">
   <property name="text">
    <string>Revenue per Supplier</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
